### PR TITLE
Added entry for "macOS"

### PR DESCRIPTION
**Pull request by sffrench**
Added entry for "macOS"
[Issue #68](https://github.com/StyleGuides/WritingStyleGuide/issues/68#issuecomment-702484041)

### CODE REVIEW

File: en-US/M.xml
Comment (daobrien) [inline en-US/M.xml:13]: I would introduce the list of "other operating systems" by using a colon. i.e., "...other operating systems: iOS, watchOS, and tvOS." Make that little patch and feel free to merge. Thanks.

**Review by daobrien — APPROVED** [review]
One minor nit to fix then good to go.

File: en-US/M.xml
Comment (sffrench) [reply en-US/M.xml]: Thanks, David, appreciate the feedback. Added colon to the list.

**Comment by sffrench** [comment]
> One minor nit to fix then good to go.

Quick question on the process for this repo, once my PR's are approved should I merge them myself or do you prefer to do that?

**Comment by daobrien** [comment]
> > One minor nit to fix then good to go.
> 
> Quick question on the process for this repo, once my PR's are approved should I merge them myself or do you prefer to do that?

Yes, I'd like to follow this process as much as possible. You create the PR, someone approves, you merge. If changes are requested, then typically approval is required after that, and then you merge. The exception is, as in this case, "if you fix this minor thing then good to go" = "just fix this then merge" - meaning you don't need to seek further approval. Hope that makes sense.

**Pull request merged**
